src/main/java/core/basesyntax/model/Color.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public abstract class Color implements Figure, Draw {
```

Comment:
Avoid using verbs for class/interface names. If 'Draw' is an interface, consider renaming it to something like 'Drawable' to better represent its purpose.